pro/src/context/IndividualOfferContext/IndividualOfferContext.tsx

```diff
@@ -117,12 +120,15 @@ export function IndividualOfferContextProvider({
         setSubCategories(response.payload.categoriesData.subCategories)
         setOffererNames(response.payload.offererNames)
         setVenueList(response.payload.venueList)
-        const venuesPopinDisplaying: Record<string, boolean> = {}
-        response.payload.venueList.forEach((v) => {
-          venuesPopinDisplaying[v.id] =
-            !v.hasCreatedOffer && v.hasMissingReimbursementPoint
-        })
-        setShowVenuePopin(venuesPopinDisplaying)
+
+        if (!isNewBankDetailsJourneyEnabled) {
```

Comment:
je suis un peu embrouillé mais c'est pas quand c'est actif que tu veux faire ça ?
```suggestion
        if (isNewBankDetailsJourneyEnabled) {
```

Reply:
Celui-là non car c'est l'ancien comportement, il va disparaître